### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by abdelrhmanyosry** [pimcore/pimcore#15743] [Q&A / Support] [answered]
### Pimcore version

11

### Steps to reproduce

just login the admin page and u will see the error

### Actual Behavior

When i login in the admin page i got this message and FE is broken like the icons don't appear in side bar 
this problem happened after update from pimcore 10.5 to pimcore 11

Timestamp: Fri Aug 11 2023 20:16:47 GMT+0300 (Eastern European Summer Time)
Status: 500 | Internal Server Error
URL: /admin/object/tree-get-root
Message: An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type' in 'field list'
Trace: 
in /var/www/pimcore/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:67
#0 /var/www/pimcore/vendor/doctrine/dbal/src/Connection.php(1935): Doctrine\DBAL\Driver\API\MySQL\ExceptionConverter->convert()
#1 /var/www/pimcore/vendor/doctrine/dbal/src/Connection.php(1878): Doctrine\DBAL\Connection->handleDriverException()
#2 /var/www/pimcore/vendor/doctrine/dbal/src/Connection.php(1103): Doctrine\DBAL\Connection->convertExceptionDuringQuery()
#3 /var/www/pimcore/vendor/doctrine/dbal/src/Connection.php(585): Doctrine\DBAL\Connection->executeQuery()
#4 /var/www/pimcore/vendor/pimcore/pimcore/models/DataObject/AbstractObject/Dao.php(427): Doctrine\DBAL\Connection->fetchAssociative()
#5 /var/www/pimcore/vendor/pimcore/pimcore/models/DataObject/AbstractObject.php(242): Pimcore\Model\DataObject\AbstractObject\Dao->getTypeById()
#6 /var/www/pimcore/vendor/pimcore/pimcore/models/Element/Service.php(449): Pimcore\Model\DataObject\AbstractObject::getById()
#7 /var/www/pimcore/vendor/pimcore/admin-ui-classic-bundle/src/Controller/Admin/ElementControllerBase.php(70): Pimcore\Model\Element\Service::getElementById()
#8 /var/www/pimcore/vendor/symfony/http-kernel/HttpKernel.php(166): Pimcore\Bundle\AdminBundle\Controller\Admin\ElementControllerBase->treeGetRootAction()
#9 /var/www/pimcore/vendor/symfony/http-kernel/HttpKernel.php(74): Symfony\Component\HttpKernel\HttpKernel->handleRaw()
#10 /var/www/pimcore/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle()
#11 /var/www/pimcore/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle()
#12 /var/www/pimcore/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#13 /var/www/pimcore/public/index.php(19): require_once('...')
#14 {main}

### Expected Behavior

it should start the admin page normally

**Comment by dvesh3** [comment, marked as answer]
Seems like an issue with some migrations not executed successfully. please check with command `bin/console doctrine:migrations:list --prefix=Pimcore\\Bundle\\CoreBundle` if all migrations are executed

**Reply by abdelrhmanyosry** [reply]
thank you! yeah it was related with migration

**Comment by mohammed-mahmoud-scandi** [comment]
and what about icons issue @dvesh3 
https://i.imgur.com/IuFbW5H.png

**Reply by dvesh3** [reply]
please check permissions on `/public` folder and try running `php bin/console assets:install public --symlink --relative`

**Comment by mohammed-mahmoud-scandi** [comment]
worked, Thanks
